.gitignore

```diff
@@ -195,7 +195,6 @@ gen
 .mtj.tmp/
 
 # Package Files #
-*.jar
```

Comment:
I'm going to update this to specifically unignore the wrapper jar instead of blanket-including any jar

Reply:
Sounds good!